docs/open-api/lineage.yaml

```diff
@@ -0,0 +1,281 @@
+        - $ref: '#/components/schemas/BaseFacet'
+    Run:
+      type: object
+      properties:
+        runId:
+          description: The globally unique ID of the run associated with the job.
+          type: string
+          format: uuid
+        facets:
+          description: The run facets.
+          type: object
+          anyOf:
+            - type: object
+              additionalProperties:
+                $ref: '#/components/schemas/RunFacet'
+      required:
+        - runId
+    JobFacet:
+      description: A Job Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+        - type: object
+          properties:
+            _deleted:
+              description: set to true to delete a facet
+              type: boolean
+    Job:
+      type: object
+      properties:
+        namespace:
+          description: The namespace containing that job
+          type: string
+          example: my-scheduler-namespace
+        name:
+          description: The unique name for that job within that namespace
+          type: string
+          example: myjob.mytask
+        facets:
+          description: The job facets.
+          type: object
+          anyOf:
+            - type: object
+              additionalProperties:
+                $ref: '#/components/schemas/JobFacet'
+      required:
+        - namespace
+        - name
+    DatasetFacet:
+      description: A Dataset Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+        - type: object
+          properties:
+            _deleted:
+              description: set to true to delete a facet
+              type: boolean
+    Dataset:
+      type: object
+      properties:
+        namespace:
+          description: The namespace containing that dataset
+          type: string
+          example: my-datasource-namespace
+        name:
+          description: The unique name for that dataset within that namespace
+          type: string
+          example: instance.schema.table
+        facets:
+          description: The facets for this dataset
+          type: object
+          anyOf:
+            - type: object
+              additionalProperties:
+                $ref: '#/components/schemas/DatasetFacet'
+      required:
+        - namespace
+        - name
+    InputDatasetFacet:
+      description: An Input Dataset Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+    InputDataset:
+      description: An input dataset
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Dataset'
+        - type: object
+          properties:
+            inputFacets:
+              description: The input facets for this dataset.
+              type: object
+              anyOf:
+                - type: object
+                  additionalProperties:
+                    $ref: '#/components/schemas/InputDatasetFacet'
+    OutputDatasetFacet:
```

Comment:
The difference between "Input..." and "Output..." is about the usage of a type.
These two types are actually identical.
I don't see a reason to have duplicated types in the API for this purpose.